spring/src/main/java/org/axonframework/spring/config/EventHandlerRegistrar.java

```diff
@@ -38,6 +38,7 @@
  * @author Allard Buijze
  * @since 3.0
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.